README.md

```diff
@@ -129,6 +131,7 @@ wrapperUpgrade {
 | `options.gitCommitExtraArgs` | List of additional git commit arguments                                                                                                                          |
 | `options.allowPreRelease`    | Boolean: true will get the latest Maven/Gradle version even if it's a pre-release. Default is false.                                                             |
 | `options.labels`             | Optional list of label (names) that will be added to the PR.                                                                                                     |
+| `options.ignoreClosedPRs`    | Boolean: true will recreate the PR if a closed PR with the same branch name exists. false will not create the PR. Default is false.                              |
```

Comment:
Please adapt all occurrences. I'd also use no abbreviation for `PR` in user-facing  parts. Can you adapt this, too?
```suggestion
| `options.ignoreClosedPullRequests`    | Boolean: true will recreate the pull request if a closed pull request with the same branch name exists. false will not create the pull request. Default is false.                              |
```

Reply:
done